request/request.js

```diff
@@ -2,6 +2,8 @@
 
 var buildQueryString = require("../querystring/build")
 
+var FILE_PROTOCOL_REGEX = new RegExp('^file://', 'i')
```

Comment:
That line only runs once, at load time, so it does not matter for perf.

However, a literal `/file:\/\//i` would shave a few bytes which is always nice.